modules/biobambam/bammarkduplicates2/main.nf

```diff
@@ -0,0 +1,35 @@
+process BIOBAMBAM_BAMMARKDUPLICATES2 {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::biobambam=2.0.182" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/biobambam:2.0.182--h7d875b9_0':
+        'quay.io/biocontainers/biobambam:2.0.182--h7d875b9_0' }"
+
+    input:
+    tuple val(meta), path(bam)
+
+    output:
+    tuple val(meta), path("*.bam")        , emit: bam
+    tuple val(meta), path("*.metrics.txt"), emit: metrics
+    path "versions.yml"                   , emit: versions
+
+    script:
```

Comment:
```suggestion

    when:
    task.ext.when == null || task.ext.when

    script:
```

Reply:
This should fix the linting :)

Reply:
This was added in https://github.com/nf-core/modules/pull/1261 and the linting will fail if it isn't present in the module.